### PR TITLE
Update operations-playbook.md

### PR DESCRIPTION
**Pull request by TQuock**
To update with a link to all playbooks and processes

### CODE REVIEW

**Review by TQuock — COMMENTED** [review]
looks good to me

File: operations/finance/operations-playbook.md [hunk 1]
```diff
@@ -1,2 +1,242 @@
...
+### Revenue and acounts receivable
+Internal processes and framework for order processinng, sales order review and invoicing, recognizing US GAAP revenue, cash collections, and uncollectible receivables.
+
+|Process Description/Doc Link|System/Tool|Document Type|Flow Chart|Occurrence|AOR|Access Rights|
+|:----|:----|:----|:----|:----|:----|:----|
+|[Lead to contract walkthrough](https://docs.google.com/document/d/1DucB1CJqKqucLTw_9eqsjm6Fo7mE0J1vtaYcP_2Q_as/edit)|`SFDC`|Process Narrative| |Daily|Ben Graham|Mattermost|
+|[Order management training videos](https://drive.google.com/drive/folders/1fgpSWCvJ0G97iSngvQr4UlTrv6xatzu7?usp=sharing)|`SFDC`|Drive| |Daily|Nikki Johnson, Jeff Dynda|Accounting|
+|[Order management resources](https://drive.google.com/drive/folders/1qYOLgNIFPXxE4Kl875OJWYjv5wHMxlpj?usp=sharing)|`SFDC`|Drive| |Daily|Nikki Johnson, Jeff Dynda|Accounting|
+|[Non-production key license provisioning](https://docs.google.com/document/d/1eSVIB40SpEaTHbjd2E9Sl8jt9j0U_Ngjy-aXnQsvbIY/edit?usp=sharing)|`Netsuite`|Playbook| |As needed|Jeff Dynda|Accounting|
+|[True-up customer seats]|`SFDC`|Playbook| |Quarterly|Jeff Dynda|Accounting|
```
Comment (jasonblais) [inline operations/finance/operations-playbook.md:53]: @TQuock Is there a link to true-up customer seats you'd like to include here?

File: operations/finance/operations-playbook.md [hunk 2]
```diff
@@ -1,2 +1,242 @@
...
+|[Divvy virtual credit card walkthrough](https://docs.google.com/document/d/1Yypp0jbciDspLDp1_bx0aKi7EuCLs9I6ppc2-zGm6zM/edit#heading=h.hsezwk8fw18o)|`Divvy` `Netsuite`|Process Narrative| |Weekly|Ben Graham|Accounting|
+|[Issue, book, monitor, and reconcile divvy corporate cards](https://docs.google.com/document/d/1kLrTfBqeWyPyXia17LJwLn-vI31j2uMJbs_DEd46p7E/edit#heading=h.t8lxnw6fufgs)|`Divvy` `Netsuite` `Accrualify`|Playbook| |Weekly|Ben Graham|Accounting|
+|[Reconciling cash](https://docs.google.com/document/d/1igoLD-tSKXqukAcQkanjHDz2FJGp_CT9pOIlaovn3kM/edit#heading=h.3gt7174pkqjt)|`Netsuite`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Manage and reconcile Paypal](https://docs.google.com/document/d/1FibtBTryBYWSxKEoLULnXbGNK8NjiUjBHhzdXZPQAeM/edit#heading=h.qz5k1zqf0i4s)|`Netsuite` `Paypal`|Playbook| |Weekly|Jeff Dynda|Accounting|
+|[Cash disbursement waterfall](https://docs.google.com/document/d/1hpD-c16kx_lliXhuxkF9oNZED9idlzk2O-0nJQlptbY/edit#)|`Netsuite`|Playbook| |Monthly|Jeff Dynda|Accounting|
+
+
+### Revenue and acounts receivable
```
Comment (jasonblais) [inline operations/finance/operations-playbook.md]:
```suggestion
### Revenue and accounts receivable
```

File: operations/finance/operations-playbook.md [hunk 3]
```diff
@@ -1,2 +1,242 @@
...
+
+
+
+### Fixed assets
+Internal processes and framework for booking and recognizing phyiscal property and depreciation/amortization. 
+
+|Process Description/Doc Link|System/Tool|Document Type|Flow Chart|Occurrence|AOR|Access Rights|
+|:----|:----|:----|:----|:----|:----|:----|
+|[Fixed asset walkthrough](https://docs.google.com/document/d/1Weh3sAyNBvkO0mO_4IUGAltISTE96uEn4sYov9MbtD0/edit)|`Netsuite`|Process Narrative| |Monthly|Jeff Dynda|Accounting|
+|[Mattermost hardware buyback]|`Netsuite`|Playbook| |Quarterly|Jeff Dynda|Accounting|
```
Comment (jasonblais) [inline operations/finance/operations-playbook.md:84]: @TQuock same here, any specific link you want to include for hardware buyback?

File: operations/finance/operations-playbook.md (hunk 1 above)
Comment (TQuock) [reply operations/finance/operations-playbook.md:53]: Thanks for highlighting, Jason! These are placeholders for now as we finalize the documentation. More PRs to come in the next few weeks.

File: operations/finance/operations-playbook.md (hunk 3 above)
Comment (TQuock) [reply operations/finance/operations-playbook.md:84]: Thanks for highlighting, Jason! These are placeholders for now as we finalize the documentation. More PRs to come in the next few weeks.

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@justinegeffen NOT urgent - would you be open to help with an editor review next week post-merge? I did a quick review but likely missed things. Will be good to have a quick review from you as well.

**Pull request merged**